index.js

```diff
@@ -1,21 +1,41 @@
 'use strict';
 
 const config = require('./config');
-const amqp = require('amqp-wrapper')(config.amqp);
-const knex = require('knex')(config.db.knex);
 const debug = require('debug')('amqp-log-to-pg');
 
-knex.schema.createTableIfNotExists(config.db.tableName, function (table) {
-  table.increments();
-  table.jsonb('data');
-  table.timestamps(true, true);
-}).then(function () {
-  amqp.connect().then(function () {
-    amqp.consume(onMessage);
-  }).catch(console.error);
-});
+let hasMeta;
+function migrate (knex) {
+  return knex.schema.createTableIfNotExists(config.db.tableName, function (table) {
+    table.increments();
+    table.jsonb('meta');
+    table.jsonb('data');
+    table.timestamps(true, true);
+  }).then(() => {
+    return knex.schema.hasColumn(config.db.tableName, 'meta');
+  });
+}
+
+function main () {
+  const knex = require('knex')(config.db.knex);
+  migrate(knex).then(function (result) {
```

Comment:
what happens when migrate throws an error? should you be sending that to `console.error` as well rather than relying on the unhandled error exception process?

would this be nicer using async/await?